src/rsa/public/modulus.rs

```diff
@@ -0,0 +1,69 @@
+use crate::{arithmetic::bigint, bits, error, rsa::N};
+use core::ops::RangeInclusive;
+
+pub struct Modulus {
+    value: bigint::Modulus<N>,
+    bits: bits::BitLength,
+}
+
+impl core::fmt::Debug for Modulus {
+    fn fmt(&self, fmt: &mut ::core::fmt::Formatter) -> Result<(), ::core::fmt::Error> {
+        self.value.fmt(fmt)
+    }
+}
```

Comment:
I'm not too worried about the coverage failure here. 5 or so PRs from now, we can format the debug output the way we want, but we can't do that now. At that future point in time, we can add a test for this.